include/functions.h

```diff
@@ -1958,13 +1958,13 @@ void Interface_ChangeAlpha(u16 param_1);
 // void func_80111CB4(void);
 // void func_801129E4(void);
 void func_80112AFC(GlobalContext* globalCtx);
-void func_80112B40(GlobalContext* globalCtx, s32 arg1);
+void func_80112B40(GlobalContext* globalCtx, u8 arg1);
 // void func_80112BE4(void);
 // void func_80112C0C(void);
 u32 Item_Give(GlobalContext* globalCtx, u8 param_2);
 // void func_801143CC(void);
 UNK_TYPE func_80114978(UNK_TYPE arg0);
-void func_801149A0(s32 arg0, s16 arg1);
+void func_801149A0(s32 itemId, s16 itemSlotId);
```

Comment:
```suggestion
void func_801149A0(s32 itemId, s16 slotId);
```
I think `slotId` would be fine here?